<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionStatistics</class>
 <widget class="QWidget" name="ConnectionStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectionStatistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="missingConnectionLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connection has been closed or the application has terminated!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="reqOutQGroupBox">
      <property name="title">
       <string>Request out queue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="reqOutQTableWidget">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <column>
          <property name="text">
           <string># Pushed</string>
          </property>
          <property name="toolTip">
           <string>Number of requests that have been pushed on to the request out queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Overflow</string>
          </property>
          <property name="toolTip">
           <string>Number of overflows that have occurred when sending a request.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Dispatched</string>
          </property>
          <property name="toolTip">
           <string>Number of requests that have been dispatched by dose_main (i.e. have been sent onwards).</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Resp</string>
          </property>
          <property name="toolTip">
           <string>Number of responses that have been dispatched back by dose_main.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Resp to app</string>
          </property>
          <property name="toolTip">
           <string>Number of responses that have been received by the application.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Timeouts</string>
          </property>
          <property name="toolTip">
           <string>Number of timeouts that have occurred.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Capacity</string>
          </property>
          <property name="toolTip">
           <string>Maximum size of the request out queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
          <property name="toolTip">
           <string>Current size of the request out queue (i.e. number of requests that are currently in transit).</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="reqInQGroupBox">
      <property name="title">
       <string>Request in queues</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="reqInQTableWidget">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <column>
          <property name="text">
           <string>Consumer</string>
          </property>
          <property name="toolTip">
           <string>Address of the consumer that the request in queue belongs to.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Pushed</string>
          </property>
          <property name="toolTip">
           <string>Number of requests that have been pushed on to the in queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Overflow</string>
          </property>
          <property name="toolTip">
           <string>Number of overflows when pushing requests on to the in queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Dispatched</string>
          </property>
          <property name="toolTip">
           <string>Number of requests that have been dispatched to the application.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Resp</string>
          </property>
          <property name="toolTip">
           <string>Number of responses that the application has put back in the queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Resp to dose_main</string>
          </property>
          <property name="toolTip">
           <string>Number of responses dispatched by dose_main onwards.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Capacity</string>
          </property>
          <property name="toolTip">
           <string>Maximum size of the request in queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
          <property name="toolTip">
           <string>Current size of the request in queue.</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="msgOutQGroupBox">
      <property name="title">
       <string>Message out queue</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="msgOutQTableWidget">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string># Pushed</string>
          </property>
          <property name="toolTip">
           <string>Number of messages pushed on to the message out queue by the application.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Overflow</string>
          </property>
          <property name="toolTip">
           <string>Number of overflows that have happened.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Capacity</string>
          </property>
          <property name="toolTip">
           <string>Maximum size of the message out queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
          <property name="toolTip">
           <string>Current size of the message out queue.</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="msgInQGroupBox">
      <property name="title">
       <string>Message in queues</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="msgInQTableWidget">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <column>
          <property name="text">
           <string>Consumer</string>
          </property>
          <property name="toolTip">
           <string>Address of the consumer that the queue belongs to.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Pushed</string>
          </property>
          <property name="toolTip">
           <string>Number of messages that dose_main has pushed on to the queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Overflow</string>
          </property>
          <property name="toolTip">
           <string>Number of overflows that have occurred when dose_main has pushed messages on to the queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Capacity</string>
          </property>
          <property name="toolTip">
           <string>Maximum size of the message in queue.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
          <property name="toolTip">
           <string>Current size of the queue.</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
